internal/distro/fedora/distro.go

```diff
@@ -706,295 +734,186 @@ func newDistro(name, modulePlatformID, ostreeRef string) distro.Distro {
 			"dracut-config-generic",
 			"grub2-pc",
 		},
-		buildPackages: []string{
-			"grub2-pc",
-		},
-		legacy: "i386-pc",
+		legacy:   "i386-pc",
+		bootType: distro.LegacyBootType,
 	}
 
 	aarch64 := architecture{
 		distro: &r,
 		name:   distro.Aarch64ArchName,
 		bootloaderPackages: []string{
-			"dracut-config-generic",
-			"efibootmgr",
-			"grub2-efi-aa64",
-			"grub2-tools",
-			"shim-aa64",
-		},
-		uefi: true,
+			"dracut-config-generic", "efibootmgr", "grub2-efi-aa64",
+			"grub2-tools", "shim-aa64"},
+		bootType: distro.UEFIBootType,
 	}
 
+	iotServices := []string{
+		"NetworkManager.service", "firewalld.service", "rngd.service", "sshd.service",
+		"zezere_ignition.timer", "zezere_ignition_banner.service",
+		"greenboot-grub2-set-counter", "greenboot-grub2-set-success", "greenboot-healthcheck", "greenboot-rpm-ostree-grub2-check-fallback",
+		"greenboot-status", "greenboot-task-runner", "redboot-auto-reboot", "redboot-task-runner",
+		"parsec", "dbus-parsec"}
```

Comment:
Please use one service name on a line. This will make it easier to review change diffs in the future. Thanks.